com.google.Chromium.yaml

```diff
@@ -0,0 +1,222 @@
+app-id: com.google.Chromium
```

Comment:
is this the app-id used by the app? Given that chromium committers are very visible with their @chromium.org address' ;-)

Reply:
Agreed. I'd probably try `org.chromium.Chromium` if I wanted to install this

Reply:
Given how irritating it is to change app-ids once installed it would be ideal to have an upstream ruling/evidence of existing usage. May be impractical of course, but ideal.

Reply:
Whoops, I didn't think about that at all. org.chromium.Chromium is definitely more correct and is what is used for the plist files on macOS, so I've renamed it.